### PR TITLE
Hl7 2 ajp upload file to server

### PR DESCRIPTION
**Pull request by Perry5**
Ticket: https://jira.amida.com/browse/HL7-2

## What does this ticket do?
- Adds endpoint to upload a file (using multer)
- Moves sample HL7 messages to `/data` folder
- Removes JWT requirement from config file
- Adds unit test for upload function
- Note that this doesn't do any validation of the file that is uploaded, that is a separate [ticket](https://jira.amida.com/browse/HL7-11). This only uploads the file and saves it. It was mainly to create a template so other people can work around it. 

## How to Test:
 -  Send a POST request from postman with a file in the request body
 - Make sure the file gets uploaded to `data/hl7-uploads`
 - Make sure your mongoDB has the name of the file and the date/time it was uploaded
- Make sure `yarn test` passes and there are no linting errors.

### CODE REVIEW

File: package.json
Comment (orndorffgrant) [inline package.json]: Run `yarn config set save-prefix ""` so that yarn won't add `false` before version numbers

File: data/hl7-uploads/500HL7Messages.txt-1561400237743
Comment (orndorffgrant) [inline data/hl7-uploads/500HL7Messages.txt-1561400237743]: I don't think the files in `hl7-uploads` should be stored in git

File: server/hl7/hl7.controller.js
Comment (orndorffgrant) [inline server/hl7/hl7.controller.js:8]: this adds the date after the file extension, i.e. `originalfilename.txt-123456789`, not a big deal, but we probably want the file extension to stay at the end of the name

File: server/hl7/hl7.route.js
Comment (orndorffgrant) [inline server/hl7/hl7.route.js:8]: this is cool

**Review by orndorffgrant — CHANGES_REQUESTED** [review]
Looks good! Couple small requests after code review. Functional review is next on my ToDo.

File: data/hl7-uploads/500HL7Messages.txt-1561400237743
Comment (Perry5) [reply data/hl7-uploads/500HL7Messages.txt-1561400237743]: Good point- let me fix that. Oversight on my part. Let me fix that

File: server/hl7/hl7.controller.js
Comment (Perry5) [reply server/hl7/hl7.controller.js:8]: Yes, this is intentional for now. Just wanted to create something for other to work from. Changes to this will be coming in another PR.

**Comment by Perry5** [comment]
@orndorffgrant updated

**Review by orndorffgrant — APPROVED** [review]
Added back the hl7 sample data and kept the uploads directory without the uploads themselves.
Also added debug config for vscode.
Good to go!

**Comment by Perry5** [comment]
Rule of thumb is to have 2 approvers before we merge in a PR. However because this PR handles things like removing JWT and such, i'm going to merge it in so we are not dealing with those annoying issues.

**Pull request merged**
